data/en/test_radio_mandatory.json

```diff
@@ -0,0 +1,43 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "radio",
+        "title": "RadioMandatory",
```

Comment:
The title should include spaces, as otherwise it can be confused with being an enumerated value e.g. RepeatingAnswer